docs/shared/reference/nx-json.md

```diff
@@ -1,10 +1,10 @@
 # nx.json
 
-The `nx.json` file configures the Nx CLI and project defaults.
+The `nx.json` file configures the Nx CLI and project defaults. The full [machine readable schema](https://github.com/nrwl/nx/blob/master/packages/nx/schemas/nx-schema.json) is available on Github.
 
-The following is an expanded version showing all options. Your `nx.json` will likely be much shorter.
+The following is an expanded example showing all options. Your `nx.json` will likely be much shorter. For a more intuitive understanding of the roles of each option, you can highlight the options in the excerpt below that relate to [task orchestration](#Orchestration), [task execution](#Execution) and [task caching](#Caching).
```

Comment:
Oh, the ability to link to highlighted code is interesting!

Can we make this stand out more? Like have something like "Click to highlight the corresponding section: <links here...>"? Because initially when reading through this these links looked like links to further reading, so I didn't even click them.

I was only trying to understand why the highlighting didn't show up for me which is when I found it.

Alternatively keep this button always visible and not just on hover
![image](https://github.com/nrwl/nx/assets/542458/773f4356-eee4-4c83-8531-6d664a1fd291)

which IMO would make a lot of sense